.circleci/config.yml

```diff
@@ -115,7 +117,9 @@ commands:
       - run: make lint
       - run: make check
       - run: make integration
-      - run: make test
+      - run:
+          command: make test
+          no_output_timeout: 15m
```

Comment:
`go test` times out after 10 minutes. Change the CircleCI timeout to something larger than that so that the timeout error is properly reported.